scripts/us_cdc/500_places/parse_cdc_places.py

```diff
@@ -0,0 +1,349 @@
+    },
+                inplace=True)
+    data = data.drop(columns=[
+        "Measure", "Category", "DataSource", "Data_Value_Type", "StateAbbr",
+        "StateDesc", "Data_Value_Unit", "Data_Value_Footnote_Symbol",
+        "Data_Value_Footnote", "Geolocation", "LocationName", "CategoryID",
+        "Short_Question_Text"
+    ])
+    return data
+
+
+def clean_zip_code_data(data):
+    """
+    Args:
+        data: pandas dataframe with zip code-level data to be cleaned
+    Returns:
+        a dataframe with cleaned zip code-level data
+    """
+    data["LocationID"] = data["LocationID"].apply(generate_zip_code_dcids)
+    data.rename(columns={
+        'MeasureId': 'StatVar',
+        'LocationID': 'Location'
+    },
+                inplace=True)
+    data = data.drop(columns=[
+        "Measure", "Category", "DataSource", "Data_Value_Type",
+        "Data_Value_Unit", "Data_Value_Footnote_Symbol", "Data_Value_Footnote",
+        "Geolocation", "LocationName", "CategoryID", "Short_Question_Text"
+    ])
+    return data
+
+
+def generate_statvar_names(data):
+    """
+    Args:
+        data: pandas dataframe with data to be cleaned
+    Returns:
+        a dataframe with additional columns of StatVar names
+    """
+    data[
+        "Low_Confidence_Limit_StatVar"] = "dcs:LowerConfidenceIntervalLimit_" + data[
+            "MeasureId"].map(MEASURE_TO_STATVAR_MAP)
+    data[
+        "High_Confidence_Limit_StatVar"] = "dcs:UpperConfidenceIntervalLimit_" + data[
+            "MeasureId"].map(MEASURE_TO_STATVAR_MAP)
+    data["Population_StatVar"] = "dcs:SampleSize_" + data["MeasureId"].map(
+        MEASURE_TO_STATVAR_MAP)
+    data["MeasureId"] = "dcs:" + data["MeasureId"].map(MEASURE_TO_STATVAR_MAP)
+    data["DataValueTypeID"] = "dcs:" + data["DataValueTypeID"].map(
+        DATA_VALUE_TYPE_MAP)
+    data["ConfidenceLevelTypeID"] = data["DataValueTypeID"] + "ConfidenceLevel"
+    return data
+
+
+def clean_cdc_places_data(input_file, output_file, sep):
+    """
+    Args:
+        input_file: path to a comma-separated CDC Places data file
+        output_file: path for the cleaned csv to be stored
+    Returns:
+        a cleaned csv file
+    """
+    print("Cleaning file...")
+    data = pd.read_csv(input_file, sep=sep)
+    data = generate_statvar_names(data)
+    if "tract" in input_file:
+        data = clean_census_tract_data(data)
+    elif "county" in input_file:
+        data = clean_county_data(data)
+    elif "city" in input_file:
+        data = clean_city_data(data)
+    elif "zip" in input_file:
+        data = clean_zip_code_data(data)
+    data = data.replace(np.nan, '', regex=True)
+    print("Writing to output file...")
+    data.to_csv(output_file, index=False)
+    print("Finished cleaning file!")
+
+
+def main():
+    """Main function to generate the cleaned csv file."""
+    sep = ","
+    input_file = sys.argv[1]
+    output_file = sys.argv[2]
+    if len(sys.argv) > 3:
+        sep = sys.argv[3].strip('sep=')
```

Comment:
Document the 3rd argument?

An easier/cleaner approach to dealing with flags is the absl library (e.g., https://github.com/datacommonsorg/data/blob/master/scripts/us_eia/opendata/download_bulk.py#L27-L34)

Reply:
added documentation for the 3rd argument in the file header. Will look into the absl library for future imports, thanks!